src/scss/_variables.scss

```
@@ -0,0 +1,20 @@
+$blue:    #4072bd;
+$indigo:  #532896;
+$purple:  #705e93;
+$pink:    #852756;
+$red:     #6d1921;
+$orange:  #91582a;
+$yellow:  #c2a85a;
+$green:   #22db31;
```

I think we should use a different shade of green for the site.
```suggestion
$green:   #179922;
```

---

Awesome green 10/10